.github/workflows/ansible-test.yml

```diff
@@ -0,0 +1,26 @@
+name: CI
+on:
+  - pull_request
+
+jobs:
+  integration:
```

Comment:
Any reason `sanity` isn't here? If it's broken/difficult I'd be interested in knowing, and maybe able to help fix.

Reply:
@gundalow - Ah, just noticed there are some sanity tests run on k8s, I missed those initially. I thought they were skipped for the k8s modules but obviously not :P

Reply:
Got sanity tests added. Though I'm getting a few failures that ansible/ansible doesn't seem to get. Will have to look into those but will do so separately.